src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/PageType.php

```diff
@@ -37,27 +37,84 @@
+            ])
+            ->add('display_last_quantities', IntegerType::class, [
+                'label' => $this->trans(
+                    'Display remaining quantities when the quantity is lower than',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'Set to "0" to disable this feature.',
+                    'Admin.Shopparameters.Help'
+                ),
+            ])
+            ->add('display_unavailable_attributes', SwitchType::class, [
+                'label' => $this->trans(
+                    'Display unavailable attributes on the product page',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'If an attribute is not available in every product combination, it will not be displayed.',
+                    'Admin.Shopparameters.Help'
+                ),
+            ])
+            ->add('allow_add_variant_to_cart_from_listing', SwitchType::class, [
+                'label' => $this->trans(
+                    'Display the "%add_to_cart_label%" button when a product has attributes',
+                    'Admin.Shopparameters.Help',
+                    [
+                        '%add_to_cart_label%' => $this->trans(
+                            'Add to cart',
+                            'Shop.Theme.Actions'
+                        ),
+                    ]
+                ),
+                'help' => $this->trans(
+                    'Display or hide the "%add_to_cart_label%" button on category pages for products that have attributes forcing customers to see product details.',
+                    'Admin.Shopparameters.Help',
+                    [
+                        '%add_to_cart_label%' => $this->trans(
+                            'Add to cart',
+                            'Shop.Theme.Actions'
+                        ),
+                    ]
+                ),
+            ])
             ->add('attribute_anchor_separator', ChoiceType::class, [
+                'label' => $this->trans(
+                    'Separator of attribute anchor on the product links',
+                    'Admin.Shopparameters.Feature'
+                ),
                 'choices' => [
                     '-' => '-',
                     ',' => ',',
                 ],
                 'required' => true,
                 'choice_translation_domain' => 'Admin.Global',
             ])
-            ->add('display_discount_price', SwitchType::class);
+            ->add('display_discount_price', SwitchType::class, [
+                'label' => $this->trans(
+                    'Display discounted price',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'In the volume discounts board, display the new price with the applied discount instead of showing the discount (ie. "-5%").',
```

Comment:
What "discounted prices" are we talking about? Specific prices? Because I've just tested the feature and couldn't see the difference between when it's enabled and when it's disabled. 🤔 I don't understand what is the "volume discounts board".